static/styles/right-sidebar.scss

```diff
@@ -236,12 +236,13 @@
     margin-right: 10px;
 }
 
-#sidebar-keyboard-shortcuts #keyboard-icon {
+#keyboard-icon {
     position: fixed;
+    bottom: 8px;
```

Comment:
@synicalsyntax Just a suggestion, would a comment indicating that the 8px is equal to the bottom padding of `.compose-content` be helpful? I'll leave including this to your discretion.